resources/assets/components/UserOverview/index.js

```diff
@@ -61,21 +63,16 @@ class UserOverview extends React.Component {
         <div className="container__block">
           <h2 className="heading -emphasized -padded"><span>User Info</span></h2>
         </div>
+
         <div className="container__block">
-          <h2 className="heading">{name}, {calculateAge(user.birthdate)}</h2>
-          <p>
-            {user.email ? <span>{user.email}<br/></span>: null}
-            {user.mobile ? <span>{user.mobile}<br/></span> : null }
-            {cityState ? <span>{cityState}<br/></span> : null }
-          </p>
-        </div>
-        <div className="container__block">
-          <h4 className="heading">Meta</h4>
-          <p>
-            <span>Source: {user.source}<br/></span>
-            <span>Northstar ID: {user.id}<br/></span>
-          </p>
+          <UserInformation user={user} includeMeta={false}>
```

Comment:
Oh great catch! it does nothing, something left over from a previous iteration of the code.